languages/csharp/exercises/concept/nullability/Nullability.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+public static class Nullability
```

Comment:
This does go against the C# guidelines, so we'll have to fix this. But for now, we can name the class for the domain bit we are modelling:

```suggestion
public static class Badge
```